packages/atomic/src/components.d.ts

```diff
@@ -2511,6 +2515,10 @@ declare namespace LocalJSX {
           * The desired layout to use when displaying results. Layouts affect how many results to display per row and how visually distinct they are from each other.
          */
         "display"?: ResultDisplayLayout;
+        /**
+          * A list of non-default fields to include in the query results, separated by commas.
+         */
+        "fieldsToInclude"?: string;
```

Comment:
This change to `components.d.ts` implies that `fieldsToInclude` was also added to the search box component. Is that accidental?

Reply:
![Screenshot 2022-06-13 at 13 28 36](https://user-images.githubusercontent.com/30511433/173353616-67614dc7-b507-4286-9c81-16a3d36e9706.png)
![Screenshot 2022-06-13 at 13 28 48](https://user-images.githubusercontent.com/30511433/173353656-0c1f7c07-edf9-4e76-bf63-79bdfd298fcf.png)

both changes on this file are under `interface AtomicSearchBoxInstantResults {`, as far as I can tell. I think you missed lines 834 and 2509. can you confirm for me, please?? I didn't even touch the search box!